src/main/java/com/faforever/client/map/MapVaultController.java

```diff
@@ -249,6 +256,7 @@ private void openMapUploadWindow(Path path) {
   }
 
   public void onRefreshButtonClicked() {
+    JavaFxUtil.assertBackgroundThread();
```

Comment:
Why should tis run on Backgound thread

Reply:
Yeah it should not, and the assert fails so the button is actually broken. Need to remove that.